src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```diff
@@ -42,10 +42,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DotnetCliToolTargetFramework>netcoreapp2.2</DotnetCliToolTargetFramework>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <IncludeBuildOutput Condition=" '$(PackAsTool)' == 'true' ">false</IncludeBuildOutput>
-    <PackageType Condition=" '$(PackAsTool)' == 'true' ">DotnetTool</PackageType>
-    <RuntimeIdentifiers Condition=" '$(PackAsTool)' == 'true' ">$(RuntimeIdentifiers);$(PackAsToolShimRuntimeIdentifiers)</RuntimeIdentifiers>
+  <PropertyGroup Condition=" '$(PackAsTool)' == 'true' ">
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <PackageType>DotnetTool</PackageType>
+    <RuntimeIdentifiers>$(RuntimeIdentifiers);$(PackAsToolShimRuntimeIdentifiers)</RuntimeIdentifiers>
+    <RollForward Condition="'$(RollForward)' == ''">Major</RollForward>
```

Comment:
Where is the detection logic for `RollFoward`? I dont see this anywhere in the SDK repository. I dont know if `Major` is the same as `LatestMajor`?

Reply:
I think `Major` works here for `RollForward`? Can @baronfel confirm this?

Reply:
`Major` is what we want here - `LatestMajor` would _force_ the tool to run on a more recent runtime if one was available, but we want this automatic re-targeting to be a last-chance effort. If a tool was compiled using a TFM of net7.0, it was clearly _intended_ to run on the 7.0 runtime if present, and we should prefer that. A user can signal more braod support by either choosing `LatestMajor` themselves, or multitargeting the tool to target e.g. `net8.0`.